docker/fullnode-x/docker-compose.yaml

```diff
@@ -0,0 +1,61 @@
+version: "3.9"
+
+services:
+
+  fullnode:
+    build:
+      context: ../..
+      dockerfile: ./docker/sui-node/Dockerfile
+      args:
+        # update with git sha of whichever branch you're building on.
+        GIT_REVISION: db24658bc9
+        BUILD_DATE: today
+
+    environment:
+      - LD_PRELOAD
+      - RUST_JSON_LOG="true"
+      - RUST_LOG="debug"
+#      - RUST_LOG="sui_core=debug,narwhal=info,info"
```

Comment:
Maybe just remove this altogether for now?
```suggestion
```

Reply:
right. I was debugging. don't need it now. but, so far it's still catching up and not logging much.